<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonCreateDB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>데이터베이스생성</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddCustomer">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>고객추가</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddTable">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>테이블추가</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCustomerId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEnterpriseName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCeoName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditEtc">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewCustomer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
